api/routes/score_routes.py

```diff
@@ -0,0 +1,48 @@
+# flake8: noqa
+from typing import Dict
+from typing import List
+
+from db.queries.scores import get_just_score_for_application_sub_crit
+from db.queries.scores import post_just_score_for_application_sub_crit
+from db.schemas import JustScoreMetadata
+
+
+def latest_score_for_application_sub_criteria(
+    application_id: str, sub_criteria_id: str
+) -> Dict:
+    """latest_score_for_application_sub_criteria Function 
+    used by the get endpoint `/applications/{application_id}/
+    subcriterias/{subcriteria_id}/scores`.
+
+    :param application_id: The stringified application UUID.
+    :param sub_criteria_id: The stringified sub_criteria UUID.
+    :return: A dictionary.
+    """
+
+    latest_score = get_just_score_for_application_sub_crit(
+        application_id=application_id, sub_criteria_id=sub_criteria_id
+    )
+
+    return latest_score
+
+
+def post_score_for_application_sub_criteria(
+    score: int, justification: str, application_id: str, timestamp: str, 
+    sub_criteria_id: str, user_id: str
+) -> Dict:
```

Comment:
💯 I like that the kwargs are listed here - I think just putting **kwargs here hides details.

Reply:
I'm not sure if I want to hide the details? I was thinking it would be nice to see what this func takes when investigating or for debugging etc?